core/build.gradle

```diff
@@ -14,6 +14,8 @@ dependencies {
     implementation libraries.springBootActuator
     implementation libraries.springWeb
     implementation libraries.guava
+    implementation libraries.dbSchedulerSpringBootStarter
```

Comment:
Can we make this a compile-time dependency and use `@ConditonalOnClass`?

Reply:
changed in latest